Builds/CMake/RippledCore.cmake

```diff
@@ -979,6 +981,7 @@ else ()
     src/test/rpc/LedgerData_test.cpp
     src/test/rpc/LedgerRPC_test.cpp
     src/test/rpc/LedgerRequestRPC_test.cpp
+    src/test/rpc/Manifest_test.cpp
```

Comment:
I think this needs to be `ManifestRPC_test.cpp`

Reply:
@seelabs doh! :man_facepalming: fixed